src/components/structures/ContextMenu.tsx

```diff
@@ -322,10 +322,11 @@ export class ContextMenu extends React.PureComponent<IProps, IState> {
 
         const menuClasses = classNames({
             'mx_ContextualMenu': true,
-            'mx_ContextualMenu_left': !hasChevron && position.left,
-            'mx_ContextualMenu_right': !hasChevron && position.right,
-            'mx_ContextualMenu_top': !hasChevron && position.top,
-            'mx_ContextualMenu_bottom': !hasChevron && position.bottom,
+            // Defensively check for counter cases
```

Comment:
A why would be appreciated, to avoid someone removing the checks later on.

Reply:
Should be solved by proper git log, but sure, added.